configure.ac

```diff
@@ -500,56 +411,6 @@ AC_CONFIG_FILES([
     client/man/Makefile
     contrib/completion/Makefile
     contrib/Makefile
-    daemons/dnssec/Makefile
-    daemons/Makefile
-    daemons/ipa-kdb/Makefile
-    daemons/ipa-sam/Makefile
-    daemons/ipa-otpd/Makefile
-    daemons/ipa-slapi-plugins/Makefile
-    daemons/ipa-slapi-plugins/libotp/Makefile
-    daemons/ipa-slapi-plugins/ipa-cldap/Makefile
-    daemons/ipa-slapi-plugins/ipa-dns/Makefile
-    daemons/ipa-slapi-plugins/ipa-enrollment/Makefile
-    daemons/ipa-slapi-plugins/ipa-lockout/Makefile
-    daemons/ipa-slapi-plugins/ipa-otp-counter/Makefile
-    daemons/ipa-slapi-plugins/ipa-otp-lasttoken/Makefile
-    daemons/ipa-slapi-plugins/ipa-pwd-extop/Makefile
-    daemons/ipa-slapi-plugins/ipa-extdom-extop/Makefile
-    daemons/ipa-slapi-plugins/ipa-winsync/Makefile
-    daemons/ipa-slapi-plugins/ipa-version/Makefile
-    daemons/ipa-slapi-plugins/ipa-uuid/Makefile
-    daemons/ipa-slapi-plugins/ipa-modrdn/Makefile
-    daemons/ipa-slapi-plugins/ipa-sidgen/Makefile
-    daemons/ipa-slapi-plugins/ipa-range-check/Makefile
-    daemons/ipa-slapi-plugins/topology/Makefile
-    init/systemd/Makefile
-    init/tmpfilesd/Makefile
-    init/Makefile
-    install/Makefile
-    install/certmonger/Makefile
-    install/conf/Makefile
-    install/html/Makefile
-    install/migration/Makefile
-    install/share/Makefile
-    install/share/advise/Makefile
-    install/share/advise/legacy/Makefile
-    install/share/csrgen/Makefile
-    install/share/profiles/Makefile
-    install/share/schema.d/Makefile
-    install/ui/Makefile
-    install/ui/css/Makefile
-    install/ui/src/Makefile
-    install/ui/src/libs/Makefile
-    install/ui/images/Makefile
-    install/ui/build/Makefile
-    install/ui/build/dojo/Makefile
-    install/ui/build/freeipa/Makefile
-    install/tools/Makefile
-    install/tools/man/Makefile
-    install/updates/Makefile
-    install/restart_scripts/Makefile
-    install/wsgi/Makefile
-    install/oddjob/Makefile
     ipaclient/Makefile
```

Comment:
This change caused me troubles after fixing typo in `SERVER_SUBDIR`.
Why did you move them?